test/common/tests.go

```diff
@@ -49,10 +49,12 @@ var (
 				{"B06 - Verify users with no email get default email", TestDefaultUserEmail},
 				{"C01 - Verify Alerts are not pending or firing apart from DeadMansSwitch", TestIntegreatlyAlertsPendingOrFiring},
 				{"C04 - Verify Alerts exist", TestIntegreatlyAlertsExist},
-				{"E01 - Verify Grafana Route is accessible", TestGrafanaExternalRouteAccessible},
+				{"E01 - Verify Middleware Grafana Route is accessible", TestGrafanaExternalRouteAccessible},
 				{"E02 - Verify that all dashboards are installed and all the graphs are filled with data", TestDashboardsData},
-				{"E03 - Verify dashboards exist", TestIntegreatlyDashboardsExist},
+				{"E03 - Verify middleware dashboards exist", TestIntegreatlyMiddelewareDashboardsExist},
 				{"E05 - Verify Grafana Route returns dashboards", TestGrafanaExternalRouteDashboardExist},
+				{"E09 - Verify customer dashboards exist", TestIntegreatlyCustomerDashboardsExist},
+				{"E10 - Verify Customer Grafana Route is accessible", TestCustomerGrafanaExternalRouteAccessible},
```

Comment:
@laurafitzgerald e2e tests looks to be failing due to these tests because they are `managed-api` specific so might be better to put these tests as a seperate tests suite that is specific to `managed-api` only. What do you think ? :thinking: 
* https://github.com/integr8ly/integreatly-operator/blob/master/test/common/tests.go#L17

Reply:
Nicely spotted. thanks @KevFan I will make that change. :)